INSTALL.md

```diff
@@ -190,13 +190,28 @@ installed as well as [Clang 3.9 or greater](http://releases.llvm.org/download.ht
 
 #### Clear Linux
 
-On Clear Linux, you need a few extra libraries to build Alacritty. Here's a
-`swupd` command that should install all of them. If something is still found
-to be missing, please open an issue.
+On Clear Linux, a few extra steps are needed to build Alacritty. 
+
+1. Here's a `swupd` command that should install all missing dependencies. If something is still found to be missing, please open an issue.
 
 ```sh
-swupd bundle-add devpkg-expat devpkg-freetype devpkg-libxcb
+swupd bundle-add devpkg-fontconfig devpkg-libxcb
```

Comment:
we actually need freetype, are you sure that it's not required? Also some our packages do require python to build, IIRC, you can take a look on deps from other distros.

Reply:
devpkg-freetype and devpkg-expat are both included in devpkg-fontconfig.

I don't know what else in devpkg-fontconfig is needed in addition to freetype and expat, just that I needed to install the fontconfig bundle to successfully compile from source.

Python is in the desktop-apps bundle. I don't remember if that is part of the recommended installation configuration, but it could be why I did not need to install python when compiling Alacritty.

Reply:
I think that this part should be fine the way it is right now then.

Reply:
Should I make another commit reverting the change or will you just not merge this commit?